cmd/insprd/memory/interface.go

```diff
@@ -20,7 +20,7 @@ type ChannelMemory interface {
 
 // ChannelGetInterface is an interface to get channels from memory
 type ChannelGetInterface interface {
-	Get(context string, ctName string) (*meta.Channel, error)
+	Get(context string, chName string) (*meta.Channel, error)
```

Comment:
this is already the in the channel interface, it could be just name, no?

Reply:
Sure